### PR TITLE
Adjusted splash-screen & updated locals

### PR DESCRIPTION
**Pull request by Anil-kumar-Majji**

### CODE REVIEW

File: locales/hin.json [hunk 1]
```diff
@@ -131,7 +131,6 @@
       "language": "भाषा",
       "aboutInji": "इंजी के बारे में",
       "credentialRegistry": "क्रेडेंशियल रजिस्ट्री",
-      "errorMessage": "गलत यूआरएल दर्ज किया गया। कृपया आगे बढ़ने के लिए एक वैध यूआरएल दर्ज करें।",
```
Comment (vijay151096) [inline locales/hin.json:134]: We need this error message literal

File: locales/en.json [hunk 2]
```diff
@@ -188,6 +188,7 @@
     }
   },
   "GetVcModal": {
+    "retrievingId": "Retrieving ID",
```
Comment (vijay151096) [inline locales/en.json:191]: do we need to keep this as "Retrieving ID" or do we need to change to "Retrieving Card"

Comment (Anil-kumar-Majji) [reply locales/en.json:191]: This text is in the "ApplicaionID" screen. Here we can revert UIN/VID through AID which represents IDs. So I   Preferred "Reverting ID"

**Review by vijay151096 — APPROVED** [review]
lgtm

**Pull request merged**